NT-Line-team/Behave_test
# khanhvnn - Exercise Week 6_part 2: Develop test script for 2 functions of Opencart web application

## Exercise

* [Exercise-week6_part2](https://github.com/NT-Line-team/Behave_test/issues/49)

## Overview

Develop test script for below functions of Opencart web application.
```
Buy a product and checkout
Add review for product
```
## Verification Steps

1. `git checkout pr/###`
2. Check the functionality: 
- Run the files with below commands.
python ./shopping_checkout.py
python ./review_product.py
(change the variables (e,g: port, search_phone.. where neccessary)
### Docs
